packages/orchestration/scripts/fetch-chain-info.ts

```diff
@@ -28,14 +28,45 @@ const chainNames = [
   'umee',
 ];
 
+const nonIcaChainNames = [
+  'migaloo',
+  'terra',
+  'planq',
+  'coreum',
+  'haqq',
+  'evmos',
+  'pryzm',
+  'nibiru',
+  'carbon',
+  'sei',
+  'archway',
+  'doravota',
+  'persistence',
+  'injective',
+  'sifchain',
+  'quicksilver',
+  'provenance',
+  'crescent',
+  'nolus',
+  'empowerchain',
+  'kava',
+  'kujira',
+  'shido',
+  'dymension',
+  'lava',
+  'titan',
+  'beezee',
+];
+
 const client = new ChainRegistryClient({
-  chainNames,
+  chainNames: [...icaChainNames, ...nonIcaChainNames],
 });
 
 // chain info, assets and ibc data will be downloaded dynamically by invoking fetchUrls method
 await client.fetchUrls();
 
-const chainInfo = await convertChainInfo(client);
+const icaChainSet = new Set(icaChainNames);
```

Comment:
consider `icaChainNames` itself being a Set. there really is no ordering to them

Reply:
Done